azext_iot/central/_help.py

```diff
@@ -542,6 +564,31 @@ def _load_central_users_help():
             --object-id {objectId}
             --role operator
     """
+
+    helps[
+        "iot central user update"
+    ] = """
+        type: command
+        short-summary: Update a user in the application
```

Comment:
same comment as update for devices - what limits are there?

Reply:
added long summary